src/compiler/scala/tools/nsc/symtab/classfile/ClassfileParser.scala

```diff
@@ -1265,7 +1265,7 @@ abstract class ClassfileParser(reader: ReusableInstance[ReusableDataReader]) {
         val expectedUUID = new UUID(reader.readUncompressedLong(), reader.readUncompressedLong())
         val tastyUUID = new TastyHeaderUnpickler(TASTYBytes).readHeader()
         if (expectedUUID != tastyUUID) {
-          reporter.error(NoPosition, s"Tasty UUID ($tastyUUID) file did not correspond the tasty UUID ($expectedUUID) declared in the classfile $file.")
+          reporter.warning(NoPosition, s"$file is out of sync with its TASTy file. Loaded TASTy file. Try cleaning the project to fix this issue")
```

Comment:
you can't know this, but it would be better to use `loaders.warning` here, to make the warning pass through the `-Wconf` / `@nowarn` infrastructure